core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregations.java

```diff
@@ -142,41 +80,33 @@ public static InternalAggregations reduce(List<InternalAggregations> aggregation
         return new InternalAggregations(reducedAggregations);
     }
 
-    /** The fields required to write this addAggregation to xcontent */
-    static class Fields {
-        public static final String AGGREGATIONS = "aggregations";
-    }
-
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
Since we are planning to implement toXContent equivalents for the ParsedAggregations, would it make sense to pull this (and maybe toXContentInternal()) to the abstract class?